state/allwatcher_internal_test.go

```diff
@@ -973,7 +972,7 @@ func (s *allWatcherStateSuite) TestStateWatcher(c *gc.C) {
 		Arch: &arch,
 		Mem:  &mem,
 	}
-	err = m0.SetProvisioned("i-0", "bootstrap_nonce", hc)
+	err = m0.SetProvisioned(instance.Id("i-0"), "bootstrap_nonce", hc)
```

Comment:
these casts in the SetProvisioned() calls should not be needed as this code is not being changed

Reply:
Oh it was for increasing consistency. There were a few pre-existing casts and thought it would be cleaner if it were the same. Will revert